packages/toolkit/src/view/settings/user/UserProfileTab.tsx

```diff
@@ -24,6 +25,8 @@ import {
 } from "../../../lib";
 import { FormLabel } from "../FormLabel";
 import { LoadingSpin } from "../../../components";
+// @ts-ignore
```

Comment:
@iamnamananand996 Please don't use ts-ignore in our source code, this will be considered as anti-pattern